Calculator.c

```diff
@@ -0,0 +1,70 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>

```

Comment:
Make sure that you know the use of every #include and don't use unnecessary includes.